src/common/build_info.cpp.in

```diff
@@ -0,0 +1,18 @@
+#include "common/build_info.h"
+
+/*
+ * File build_info.cpp is generated using CMake. Do NOT modify it manually! Edit
+ * build_info.cpp.in file instead.
+ */
+
+std::string marian::cmake_cache() {
```

Comment:
Let's change this to `cmakeCache()`, we are moving to camelcase for function names everywhere. If there are still functions with old convention then only because they have not been touched for a while and they should be changed to camelCase eventually. This concerns all the new functions from this PR.

Reply:
Right, will fix.